pkgs/applications/graphics/zgv/png.patch

```diff
@@ -0,0 +1,47 @@
+diff -ur zgv-5.9/config.mk zgv-5.9.new/config.mk
```

Comment:
Is this your patch? It's not immediately clear to me what issue this resolves. If it's an external patch, would it be possible to fetch it instead of checking it into the repo?